common/config/rush/npm-shrinkwrap.json

```diff
@@ -44,11 +44,6 @@
           "version": "6.0.88",
           "resolved": "https://registry.npmjs.org/@types/node/-/node-6.0.88.tgz",
           "integrity": "sha512-bYDPZTX0/s1aihdjLuAgogUAT5M+TpoWChEMea2p0yOcfn5bu3k6cJb9cp6nw268XeSNIGGr+4+/8V5K6BGzLQ=="
-        },
-        "typescript": {
-          "version": "2.4.2",
```

Comment:
still trying to figure out why there was a 2.4.2 in here, and how this change removed it